code/modules/mob/living/carbon/carbon.dm

```diff
@@ -9,11 +9,13 @@
 	ADD_TRAIT(src, TRAIT_AGEUSIA, NO_TONGUE_TRAIT)
 
 	GLOB.carbon_list += src
+	/* SKYRAT EDIT REMOVAL
 	var/static/list/loc_connections = list(
 		COMSIG_CARBON_DISARM_PRESHOVE = .proc/disarm_precollide,
 		COMSIG_CARBON_DISARM_COLLIDE = .proc/disarm_collision,
```

Comment:
so how do you disarm people now

Reply:
CQC.
Aggro grab (Somehow).
Beat them up till they literally can't hold anything that isn't taped on.
Think that's all considering the other changes.